__tests__/format_type.js

```diff
@@ -42,26 +38,26 @@ test('formatType', function (t) {
     ['...number', '...[number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)'],
     ['undefined', '[undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined)']
   ].forEach(function (example) {
-    t.deepEqual(stringify(formatType(
+    expect(stringify(formatType(
```

Comment:
I'm cool with expect and most of the way through the port, so probably going to stick with this.